src/script/components/VerificationBadge/VerificationBadges.tsx

```diff
@@ -76,6 +81,32 @@ const useConversationVerificationState = (conversation: Conversation) => {
   return {MLS: mlsState, proteus: proteusVerificationState};
 };
 
+const useMLSStatus = (identity?: WireIdentity) => {
+  const [MLSStatus, setMLSStatus] = useState<MLSStatuses | undefined>(undefined);
+  const {current: userState} = useRef(container.resolve(UserState));
+
+  useEffect(() => {
+    if (!identity) {
+      setMLSStatus(undefined);
+      return;
+    }
+    void (async () => {
+      const user = await waitFor(() =>
+        userState
+          .users()
+          .find(user => stringifyQualifiedId(user.qualifiedId) === stringifyQualifiedId(identity.qualifiedUserId)),
+      );
```

Comment:
in which is the user not there?

Reply:
doesn't this create some blinking on the UI?

Reply:
I didn't experience that yet, but it theoretically could lead to that. We need to keep an eye on that